requirements.txt

```diff
@@ -1,28 +1,26 @@
-# app requirements
+#app requirements
+celery[sqs]==4.4.1
+Flask==1.1.1
+Flask-WeasyPrint==0.6
+Flask-HTTPAuth==3.3.0
+Werkzeug==1.0.0
+
+# pdf libraries
 CairoSVG==2.2.1
```

Comment:
version `2.4.2` is out now, but that version breaks svg -> png conversion

we no longer use that code path anyway - the PNG version of the logo is only used in templated letter print previews, which we have changed to now use the vector version.